src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -107,7 +110,8 @@ def create_test(
     )
     logger.info("Uploading files to test %s", test_id)
     upload_files_helper(
-        client, test_id, yaml, test_plan, load_test_config_file, not custom_no_wait
+        client, test_id, yaml, test_plan, load_test_config_file, not custom_no_wait,
+        test_type or yaml_test_body.get("kind") if yaml_test_body else response.get("kind")
```

Comment:
Can we get this in variable before calling this method

Reply:
updated